modules/base/rendering/renderablemodel.cpp

```diff
@@ -689,6 +689,8 @@ void RenderableModel::update(const UpdateData& data) {
     setBoundingSphere(_geometry->boundingRadius() * _modelScale *
         glm::compMax(data.modelTransform.scale)
     );
+    // Set Interaction sphere to be 10% in size as the bounding sphere
```

Comment:
Nitpicky, but language is weird here. Maybe "Set interaction sphere size to be 10% of the bounding sphere"?